covid19-etl/etl/idph_hospital_utilization.py

```diff
@@ -162,6 +162,7 @@ def submit_metadata(self):
         self.metadata_helper.batch_submit_records()
 
         print("Submitting summary_clinical data")
+        print(f"Length of summary_clinicals is {len(self.summary_clinicals)}")
```

Comment:
+1 to adding logs 😄 but i think this one is unnecessary because it's already printed [here](https://github.com/uc-cdis/covid19-tools/blob/1b269dc8a5850235f37e30d6dfc33eda987e1319/covid19-etl/utils/metadata_helper.py#L127)

Reply:
Done!